### PR TITLE
Fixed double link to same asset by replacing it with a site-wide vari…

### PR DESCRIPTION
**Pull request by dwai213**
…able, site.sponsors. This was added in _config.yml.

This addresses #134

### CODE REVIEW

**Comment by dwai213** [comment]
❤️  @jiangdaniel

File: _config.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ facebook_username: pierobotics
 youtube_username: pierobotics
 instagram_username: pierobotics
 host: 0.0.0.0
+sponsors: /assets/sponsors/SponsorshipLevels15-16.pdf
```
Comment (viviehn) [inline _config.yml]: I think the path has to be in quotes? Because...

File: support-pie/donate.html [hunk 2]
```diff
@@ -81,7 +81,7 @@
 					<p>We appreciate and recognize our sponsors in the form of posters, banners and logos, and t-shirts seen by thousands of members of the UC Berkeley community. In addition, sponsors are warmly welcomed and encouraged to attend the competition day.</p>
 
 
-					<p>If you would like more information about the benefits, please refer to <a href="{{ '/assets/sponsors/SponsorshipLevels15-16.pdf' | prepend: site.baseurl }}">Sponsorship Levels 2015-2016.pdf</a></p>
+					<p>If you would like more information about the benefits, please refer to <a href="{{ site.sponsors | prepend: site.baseurl }}">Sponsorship Levels 2015-2016.pdf</a></p>
```
Comment (viviehn) [inline support-pie/donate.html]: ... when I click on this, it just refreshes the page (and inspecting yields an empty href) :/

**Comment by dwai213** [comment]
🙈  It's ok!

**Pull request merged**